automatic/1password/update.ps1

```diff
@@ -1,55 +1,59 @@
-Import-Module AU
+﻿Import-Module AU
 import-module $PSScriptRoot\..\..\extensions\chocolatey-core.extension\extensions\chocolatey-core.psm1
 
 function global:au_SearchReplace {
   @{
     ".\tools\chocolateyInstall.ps1" = @{
-      "(?i)^(\s*url\s*=\s*)'.*'" = "`${1}'$($Latest.URL32)'"
-      "(?i)^(\s*checksum\s*=\s*)'.*'" = "`${1}'$($Latest.Checksum32)'"
+      "(?i)^(\s*url\s*=\s*)'.*'"          = "`${1}'$($Latest.URL32)'"
+      "(?i)^(\s*checksum\s*=\s*)'.*'"     = "`${1}'$($Latest.Checksum32)'"
       "(?i)^(\s*checksumType\s*=\s*)'.*'" = "`${1}'$($Latest.ChecksumType32)'"
+      "(?i)^(\s*silentArgs\s*=\s*)'.*'"   = "`${1}'$($Latest.SilentArgs)'"
     }
   }
 }
 
-function global:au_AfterUpdate {
+function global:au_BeforeUpdate {
```

Comment:
You can't blindly change this to `BeforeUpdate`.

This function adds and removes dependencies in the nuspec file, and this can not be done in `au_BeforeUpdate` if you want them to be saved.

Reply:
This is changed.